iopipe/report.py

```diff
@@ -29,10 +29,9 @@ def __init__(self, config, context):
         :param config: The IOpipe agent config.
         :param context: The AWS Lambda context.
         """
-        self.sent = False
         self.start_time = monotonic()
+        self.sent = False
```

Comment:
Why'd you move this?

Reply:
@pselle Just making sure `start_time` is the first thing measured/done. Gotta catch all those microseconds.

Reply:
:+1: